lib/active_remote/typecasting/boolean_typecaster.rb

```diff
@@ -0,0 +1,15 @@
+module ActiveRemote
+  module Typecasting
+    class BooleanTypecaster
+       FALSE_VALUES = ["n", "N", "no", "No", "NO", "false", "False", "FALSE", "off", "Off", "OFF", "f", "F"]
+
+      def self.call(value)
+        case value
+        when *FALSE_VALUES then false
```

Comment:
I think this is the one @abrandoned said could be more efficient if we check the non-string values first. ActiveModel handles it [a bit differently](https://github.com/rails/rails/blob/52ce6ece8c8f74064bb64e0a0b1ddd83092718e1/activemodel/lib/active_model/type/boolean.rb). Perhaps we could borrow their casting pattern, but add the additional values this supports for compatibility?